### ISSUE TITLE
[Accessibility] Keyboard focus indicator is not visible on the country flag combo box

### ISSUE THREAD

**Issue by mayukh-ms** [catamphetamine/react-phone-number-input#319]
Keyboard focus indicator is not visible on “Combo box” for the country flag.

**Package version:** 2.3.5
**OS version:** Window 10
**Browser:** Microsoft Edge 44.18362.1.0
**Screen reader:** Narrator

**Comment by catamphetamine** [comment]
Hi.
I've released v3 of the library.
See if the demo works, and if not — specify a more detailed description of the issue you're having.

**Comment by a11y1** [comment]
Yes, this is still an issue in v3. 

See this issue: https://github.com/catamphetamine/react-phone-number-input/issues/410
